source/_integrations/weather.markdown

```diff
@@ -34,3 +34,9 @@ The `weather` platform only knows the below listed conditions. The reason for th
 - 'windy'
 - 'windy-variant'
 - 'exceptional'
+
+### Forecast Weather Conditions Intervals
```

Comment:
```suggestion
## Forecast information
```

Reply:
Much better title. Updated in 3e25e8a57560e47829d5779f21f9806477a3674a